cfme/utils/template/scvmm.py

```diff
@@ -11,55 +11,69 @@ class SCVMMTemplateUpload(ProviderTemplateUpload):
 
     @property
     def library(self):
-        return self.template_upload_data.get('vhds', None)
+        return self.template_upload_data.get('vhds')
```

Comment:
Q: what if `vhds` tag missing in `data_yaml`?

Reply:
This change doesn't change any behaviour as the `None` is a default value of the second parameter of `get`. So if exception should be thrown on data missing, then @john-dupuy should  use `self.template_upload_data['vhds']`

Reply:
@jaryn is correct, I was just removing the redundant specification of `None` here, do you think this should raise an error if that `key` isn't present?

Reply:
Changed to raise an error if key "vhds" isn't present. This was done so that we don't continue and try to download the file.